pom.xml

```diff
@@ -69,18 +69,25 @@
 	</modules>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
+		<openMRSVersion>2.0.0</openMRSVersion>
```

Comment:
Because we're changing the baseline OpenMRS version, we should also bump this to the next major version (this is technically a breaking change). I.e., this module version should now be 2.0.0-SNAPSHOT.

Reply:
done